cypress/integration/pages/workProgressSpec.ts

```diff
@@ -13,26 +8,34 @@ describe("Work Progress", () => {
     context("when the type field is selected", () => {
       context("when Work Number is selected in dropdown for type", () => {
         before(() => {
-          cy.findByTestId("type").select("SGP/R&D Number");
+          search().within(() =>
+            cy.findByTestId("type").select("SGP/R&D Number")
+          );
         });
         it("shows an input text field", () => {
-          cy.get("[data-testid=valueInput]").should("have.value", "");
+          search().within(() =>
+            cy.get("[data-testid=valueInput]").should("have.value", "")
+          );
         });
       });
       context("when Status is selected in dropdown for type", () => {
```

Comment:
Could this also test that the statuses are shown in the "value" select?

Reply:
Done